index.html

```diff
@@ -122,6 +159,161 @@ <h2>Methods</h2>
       </dl>
     </section>
   </section>
+  <section id="rtcrtptransceiver-interface">
+    <h3>
+      <dfn>RTCRtpTransceiver</dfn> extensions
+    </h3>
+    <p>
+      The <code><a href="https://w3c.github.io/webrtc-pc/#dom-rtcrtptransceiver">RTCRtpTransceiver</a></code>
+      interface is defined in [[WEBRTC]]. This document extends that interface
+      by adding an additional method and attribute in order to control negotiation of
+      RTP header extensions.
+    </p>
+    <pre class="idl">
+partial dictionary RTCRtpHeaderExtensionCapability {
+  RTCRtpTransceiverDirection direction = "sendrecv";
```

Comment:
I think the implications of the direction in the context of SDP and in the context of the functionality needs to be explained.
If the direction is not included in the SDP then I would think that achieving asymmetric direction of a header extension makes more sense to do through multiple transceivers.

Reply:
I.e. piggyback on the a=direction attribute of an m= section rather than inventing a RTP header extension version of the direction that needs to be added to the SDP

Reply:
Can we leave that to the header extensions spec? I really don't want to repeat everything it has to say about direction here.

Reply:
That's fine I thought this was a new concept but it isn't.